spec.bs

```diff
@@ -865,6 +874,32 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=csp-integration>CSP Integration</h3>
```

Comment:
I think this section might be best to appear under the "Interactions with other specifications" section instead of the "HTML Integration" section.